misc/acrn-config/target/parser_lib.py

```diff
@@ -83,13 +83,20 @@ def cmd_execute(cmd):
     return res
 
 
-def handle_root_dev(line):
+def handle_block_dev(line):
     """Handle if it match root device information pattern
     :param line: one line of information which had decoded to 'ASCII'
     """
     for root_type in line.split():
         if "ext4" in root_type or "ext3" in root_type:
-            return True
+            block_type = ''
+            block_dev = line.split()[0]
+            for type_str in line.split():
+                if "TYPE=" in type_str:
+                    block_type = type_str
+
+            block_format = block_dev + " " + block_type
+            return block_format
```

Comment:
Can python return value be different types?

Reply:
The python can support it. But it is not the good coding style. I will update it.
Thanks.